packages/core/src/ScrollableGridList/index.js

```diff
@@ -117,8 +117,8 @@ ScrollableGridList.propTypes = {
   autoHide: PropTypes.bool,
   cellHeight: PropTypes.number,
   children: PropTypes.node,
-  height: PropTypes.oneOfType(PropTypes.number, PropTypes.string).isRequired, // in px
-  list: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  height: PropTypes.number,
```

Comment:
`height` can be `auto`, `2rem`, etc. hence we need to support both `number` & `string` types